.github/workflows/gradle-release.yml

```
@@ -0,0 +1,55 @@
+name: Gradle Release
+
+#on:
+#  workflow_dispatch:
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Change this to below after testing:

```suggestion
on:
  workflow_dispatch:
```